hscontrol/mapper/batcher_lockfree.go

```diff
@@ -449,6 +449,38 @@ func (b *LockFreeBatcher) MapResponseFromChange(id types.NodeID, c change.Change
 	}
 }
 
+// SendDirectUpdate sends a MapResponse directly to a specific node without going through
```

Comment:
I think my original suggestion was to batch these PR a bit bigger than one for each, so we avoid this where we have the same commits in multiple PRs.